flask_api.py

```diff
@@ -96,7 +101,7 @@ def handle_question():
     server are:
         * storage of the logs of this question-answer-session.
     """
-    initializeNimbus()
+    init_nimbus_db()
```

Comment:
Something to consider in the future: https://pythonise.com/series/learning-flask/python-before-after-request#before_request

Reply:
oh yeah, very interesting to consider!
being able to easily decorate and run code before/after a request is powerful